src/components/skills-quiz/JobDescriptions.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {
+  useEffect, useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { InfoOutline } from '@edx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+import { Skeleton } from '@edx/paragon';
+import { fetchJobPathDescription } from './data/service';
+import { DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE, JOB_DESCRIPTION_DISCLAIMER } from './constants';
+
+const JobDescriptions = ({
+  currentJobID, futureJobID, currentJobDescription, futureJobDescription, goal,
+}) => {
+  const [isLoadingJobPathDescription, setIsLoadingJobPathDescription] = useState(false);
+  const [jobPathDescription, setJobPathDescription] = useState();
+  const jobDescription = goal === DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE ? currentJobDescription : futureJobDescription;
+
+  useEffect(() => {
+    const getJobPathDescription = async () => {
+      try {
+        setIsLoadingJobPathDescription(true);
+        const description = await fetchJobPathDescription(currentJobID, futureJobID);
+        setJobPathDescription(description);
+      } catch (error) {
+        logError(error);
+      } finally {
+        setIsLoadingJobPathDescription(false);
+      }
+    };
+    if (goal !== DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE) {
+      getJobPathDescription();
+    }
+  }, [currentJobID, futureJobID, goal]);
+
+  return (
+    <div className="job-descriptions-container">
+      <div className="disclaimer mb-3">
+        <InfoOutline className="mr-1" style={{ width: '2rem', height: '2rem' }} />
+        <span className="font-italic font-weight-light">{JOB_DESCRIPTION_DISCLAIMER}</span>
+      </div>
+      <div className="job-description mb-3">{jobDescription || 'job description currently does not exist in algolia'}</div>
```

Comment:
nit: Can we rephrase this to something shorter? Like "Description not available" in a lighter tone text.

Reply:
ah... this hardcoded text was just for testing. Will remove this before merge.

Reply:
hardcoded text removed